lib/Crawler.js

```diff
@@ -45,11 +41,19 @@ function validateUrl(crawlerSeedUrl) {
  */
 function setConfigRootUrl(config, crawlerSeedUrl) {
   let portString = '';
+  let protocolString = '';
 
   if (crawlerSeedUrl.port) {
     portString = `:${crawlerSeedUrl.port}`;
   }
-  config.rootUrl = `https://${crawlerSeedUrl.hostname}${portString}`;
+
+  if (crawlerSeedUrl.protocol) {
```

Comment:
you could do this all in one line: `let protocolString = crawlerSeedUrl.protocol || 'https';

Reply:
nice